### PR TITLE
Added spec and usage to bleed

### LINKED ISSUE #1467: Bleed: Usage & Specifications
Add usage and specifications pages to Bleed component

### PR DESCRIPTION
**Pull request by Lisa159**
**Checklist**
_(Check off all the items before submitting)_
- [x] Build process is done without errors and all tests pass in `/lib` directory.
- [x] Self-reviewed the code prior to submitting.
- [x] Meets accessibility standards.
- [x] Added/updated documentation to `/website` as needed.
- [x] Added/updated tests as needed.

**Description**
Added usage and specs to bleed component.

**Closes #1467**

### CODE REVIEW

**Review by Jialecl — COMMENTED** [review]
The image bleed_specs.png seems to have multiple issues regarding the naming of the spaces.

File: website/screens/components/bleed/usage/BleedUsagePage.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { DxcFlex, DxcParagraph } from "@dxc-technology/halstack-react";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import DocFooter from "@/common/DocFooter";
+
+const sections = [
```
Comment (Jialecl) [inline website/screens/components/bleed/usage/BleedUsagePage.tsx:6]: Add the examples to usage.

Comment (Lisa159) [reply website/screens/components/bleed/usage/BleedUsagePage.tsx:6]: Removed

File: website/screens/components/bleed/usage/BleedUsagePage.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  {
+    title: "Usage",
+    content: (
+      <DxcParagraph>
+        Bleed properties refer to the scale of the spacing used outside a
+        container. The unit commonly used for the bleed scale is REM, which is a
+        measurement unit that refers to the font-size of the root element of a
+        document. The following values of REM are commonly used in the design
+        system: 0, 0.125, 0.25, 0.5, 1, 1.5, 2, 3, 4, and 5.
+      </DxcParagraph>
```
Comment (raquelarrojo) [inline website/screens/components/bleed/usage/BleedUsagePage.tsx:16]: These are the values of `space` prop, so they should be inside `Code` label.

File: website/screens/components/bleed/usage/BleedUsagePage.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  return (
+    <DxcFlex direction="column" gap="4rem">
+      <QuickNavContainerLayout>
+        <QuickNavContainer
+          sections={sections}
+          startHeadingLevel={2}
+        ></QuickNavContainer>
+      </QuickNavContainerLayout>
+      <DocFooter githubLink="https://github.com/dxc-technology/halstack-react/blob/master/website/screens/components/button/usage/BleedUsagePage.tsx" />
+    </DxcFlex>
```
Comment (raquelarrojo) [inline website/screens/components/bleed/usage/BleedUsagePage.tsx:31]: Change `button` by `bleed`.

**Review by raquelarrojo — CHANGES_REQUESTED** [review]

File: website/screens/components/bleed/usage/BleedUsagePage.tsx (hunk 2 above)
Comment (Lisa159) [reply website/screens/components/bleed/usage/BleedUsagePage.tsx:19]: Done

File: website/screens/components/bleed/usage/BleedUsagePage.tsx (hunk 3 above)
Comment (Lisa159) [reply website/screens/components/bleed/usage/BleedUsagePage.tsx:34]: Done

**Review by raquelarrojo — APPROVED** [review]

**Pull request merged**